learnr_bank/learnr_set1.Rmd

```diff
@@ -0,0 +1,386 @@
+138**2 
+#Equally valid approach
+138^2 
+```
+
+``` {r, calc-q4-answer}
+14^0.5
+```
+
+
+### Set 2: Working with logic: TRUE and FALSE
+
+1. Use a *logical operator* to ask if 56 is greater than 7.
+  
+``` {r logical_q1, exercise = TRUE}
+
+```  
+
+<div id="logical_q1-hint">
+**Hint:**  Logical operators include: `>`, `<`, `>=`, `<=`, `==`
+</div>
+
+2. Use a *logical operator* to ask if 110 is equal to 12.
+  
+``` {r logical_q2, exercise = TRUE}
+
+``` 
+  
+<div id="logical_q2-hint">
+**Hint:**  When comparing *equality* of numbers, you MUST use two equals!! `==`
+</div>
+
+3. Use a *logical operator* to ask if 32 is less than or equal to 88.
+  
+``` {r logical_q3, exercise = TRUE}
+
+``` 
+
+<div id="logical_q3-hint">
+**Hint:**  `<` is less than, but `<=` is less than *or equal to*`
+</div>
+
+4. Use a *logical operator* to ask if 100 is equal to -10 squared.
+  
+``` {r logical_q4, exercise = TRUE}
+
+``` 
+
+<div id="logical_q4-hint">
+**Hint:** Don't forget to use parentheses!!
+</div>
+
+5. Use a *logical operator* to ask if 100 is *not equal* to -55. 
+  
+``` {r logical_q5, exercise = TRUE}
+
+``` 
+
+<div id="logical_q5-hint">
+**Hint:** Check if values are *not equal* with the logical operator `!=`. Generally speaking, exclamation marks negate logical statements.
+</div>
+
+6. Use a *logical operator* to ask if 100 is *not less than* 88. 
+ 
+``` {r logical_q6, exercise = TRUE}
+
+``` 
+
+<div id="logical_q6-hint">
+**Hint:** *Negate* any other logical operator (besides `==`, which becomes `!=`), by *enclosing the statement inside parentheses prefixed by an exclamation mark, e.g. `!(...statement...)`
+</div>
+
+7. In all questions above, you wrote *single logical statements* - you used a logical operator to evaluate if a single thing was TRUE or FALSE. This time, write a *compound logical statement* to ask if: 100 is equal to 100 *and* 66 is equal to 66.
+
+``` {r logical_q7, exercise = TRUE}
+
+``` 
+
+```{r logical_q7-hint-1}
+#Combine statements with *and* using the ampersand `&`
+```
+
+```{r logical_q7-hint-2}
+#Statements with "and" will only evaluate as TRUE if *both* conditions are TRUE. `
+```
+
+8. Write a *compound logical statement* to ask if 100 is greater than to 87 *and* 66 is less than 12.
+  
+``` {r logical_q8, exercise = TRUE}
+
+``` 
+
+<div id="logical_q8-hint">
+**Hint:**  Combine statements with *and* using the ampersand `&`
+</div>
+
+9. Similar to the previous question, write a *compound logical statement* to ask if 100 is greater than to 87 *OR* 66 is less than 12.
+  
+``` {r logical_q9, exercise = TRUE}
+
+``` 
+
+```{r logical_q9-hint-1}
+#Combine statements with *or* using the "pipe operator" `|`. This is not the letter L or I - this symbol is located on the backslash `\` key. Press Shift + `\` to get it.
+```
+
+```{r logical_q9-hint-2}
+#Statements with "or" will evaluate as TRUE if *at least one* condition is TRUE. `
+```
+
+### Set 2: Answer Key
+``` {r, logical-q1-answer}
```

Comment:
This should be `logical_q1-answer`. I'm trying to use underscores for the question itself, and keep dashes reserved for the "type" of chunk it is (hint or answer). And so on for this section's answers.